db/seeds.rb

```diff
@@ -5,3 +5,45 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+require 'faker'
+require 'date'
+
+# seed for users
+
+12.times do
+  User.create!(
+    first_name: Faker::Name.neutral_first_name,
+    last_name: Faker::Name.last_name,
+    email: Faker::Internet.email,
+    password: '123456'
+    # is_owner: [true, false].sample
+  )
+end
+
+puts "You have created #{User.count} users.."
+
+# seed for costumes
+
+36.times do
+  Costume.create!(
+    name: Faker::Name.name,
+    description: "lorem ipsum",
+    animal_type: %w[dog cat].sample,
+    user_id: 1
+  )
+end
+
+puts "You have know #{Costume.count} costumes to rent.."
+
+# seed for bookings
+
+12.times do
+  Booking.create!(
+    user_id: User.ids.sample,
+    costume_id: Costume.ids.sample,
+    date_start: Date.new(2022, 7, 31),
+    date_end: Date.new(2022, 8, 1)
```

Comment:
For these two dates, find a way of making them 1) random 2) in the future and 3) a random interval (e.g. add x random days to date_start, instead of creating a new date).